CSharpOOP2/CSharpOOP2/Horse.cs

```diff
@@ -0,0 +1,156 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CSharpOOP2
+{
+    public class Horse

```

Comment:
If you want to increase your mark for this task - please complete homework with your own object/